src/chopper_count.cpp

```diff
@@ -15,29 +16,50 @@ void initialize_argument_parser(seqan3::argument_parser & parser, chopper::count
     parser.info.short_description = "Count all kmers of each file in a directory.";
     parser.info.version = "1.0.0";
 
-    parser.add_option(config.data_file, 'f', "data_file", "Give me a filename to a seqinfo file.", seqan3::option_spec::required);
-    parser.add_option(config.output_filename, 'o', "outfile", "Filename where the counts should be written to.",
+    parser.add_option(config.data_file,
+                      'f', "data_file",
```

Comment:
Do you want to remove the short ids?

Reply:
that's part of the follow up PR. I'll remove them expect for i/o

Reply:
if you want I can put it into this PR? I already worked on it

Reply:
Follow-up